tests/api/api/test_feature_sets.py

```diff
@@ -96,26 +153,78 @@ def test_feature_set_create_and_list(db: Session, client: TestClient) -> None:
     # handling multiple label queries has issues right now - needs to fix and re-run this test.
     # _assert_list_objects(client, "feature_sets", project_name, "label=owner=bob&label=color=red", 2)
 
-    # Update a feature-set
-    feature_set_update = {
-        "entities": [{"name": "market_cap", "value_type": "integer"}],
-        "labels": {"new-label": "new-value", "owner": "someone-else"},
-    }
-    response = client.put(
-        f"/api/projects/{project_name}/feature_sets/{name}/references/latest",
+
+def _perform_patch(
+    client: TestClient, project_name, name, feature_set_update, additive=False
+):
+    response = client.patch(
+        f"/api/projects/{project_name}/feature_sets/{name}/references/latest?additive={additive}",
         json=feature_set_update,
     )
     assert response.status_code == HTTPStatus.OK.value
     response = client.get(
         f"/api/projects/{project_name}/feature_sets/{name}/references/latest"
     )
-    assert response.status_code == HTTPStatus.OK.value
-    updated_resp = response.json()
+    return response.json()
+
+
+def test_feature_set_updates(db: Session, client: TestClient) -> None:
```

Comment:
replace every `update` in this function to `patch`

Reply:
I see this is repeating in other tests as well, so commenting only here, please note on the terminology, if you're testing the `update_feature_set` operation, function/variable names should be with `update_X` not `put_X`
If it's `patch_feature_set` operation - `patch_X`